specs/latest/2.0/index.html

```diff
@@ -755,7 +748,7 @@ <h2 class="no-toc">Table of contents</h2>
                   TexImageSource source); // May throw DOMException
   void texImage2D(GLenum target, GLint level, GLint internalformat, GLsizei width, GLsizei height,
                   GLint border, GLenum format, GLenum type, ArrayBufferView srcData,
-                  GLuint srcOffset);
+                  optional GLuint srcOffset = 0);
```

Comment:
Again if we're matching the nullability of these arguments from the WebGL 1.0 spec now -- this one should take "ArrayBufferView?".